src/libaktualizr/campaign/campaign_test.cc

```diff
@@ -31,6 +31,47 @@ TEST(campaign, Campaigns_from_json) {
   EXPECT_FALSE(campaignsNoAutoAccept.at(0).autoAccept);
 }
 
+/* Get JSON from campaign. */
+TEST(campaign, Campaign_to_json) {
+  auto json = Utils::parseJSONFile(test_data_dir / "campaigns_sample.json");
+
+  auto campaigns = campaign::campaignsFromJson(json);
```

Comment:
These two tests seem redundant. Furthermore, `Campaign_to_json` actually uses `campaign::campaignsFromJson`. We can probably just keep one of the two.

Reply:
Fixed. Only "Campaigns_to_json" test is remained.